easy_my_coop/models/partner.py

```diff
@@ -233,6 +233,8 @@ def get_cooperator_from_email(self, email):
         return partner
 
     def get_cooperator_from_crn(self, company_register_number):
+        if not company_register_number:
```

Comment:
@robinkeunen 

> about fixing def get_cooperator_from_crn rather than passing an empty string.

Reply:
Ha sorry, did not read carefully enough. Then it's good for me.